crucible/src/Lang/Crucible/Backend.hs

```diff
@@ -613,6 +626,59 @@ readPartExpr bak (PE p v) msg = do
   addAssertion bak (LabeledPred p (SimError loc msg))
   return v
 
+
+-- | Run the CHC solver on the current proof obligations, and return the
+-- solution as a substitution from the uninterpreted functions to their
+-- definitions.
```

Comment:
It might be worth mentioning the use of the BV<->LIA transformation somewhere in the Haddocks, since this part wasn't obvious to me until I read the source code carefully.